### ISSUE TITLE
textfield RTL not working after change language

### ISSUE THREAD

**Issue by PramodShukla10** [Abedalkareem/LanguageManager-iOS#12]

**Comment by Abedalkareem** [comment]
fixed in lates release.

**Comment by ihnatmoisieiev** [comment]
Hello @Abedalkareem,
does it work in the latest release?

**Comment by Abedalkareem** [comment]
Yes, it's working fine, but remember to keep the text alignment as natural.

**Comment by ihnatmoisieiev** [comment]
@Abedalkareem could you please check what is going on? I just added a textfield with the text alignment as natural to your example.

[LanguageManager-iOS-master.zip](https://github.com/Abedalkareem/LanguageManager-iOS/files/3583426/LanguageManager-iOS-master.zip)

**Comment by Abedalkareem** [comment]
Hi, 
When you use the natural alignment, the alignment will change depending on the language written inside the Textfiled, 
if it's Arabic the alignment will be rtl but if you use English it will be ltr.
![Simulator Screen Shot - iPhone Xs Max - 2019-09-07 at 16 18 15](https://user-images.githubusercontent.com/14274852/64474830-63691080-d18b-11e9-8c30-30705a414815.png)
![Simulator Screen Shot - iPhone Xs Max - 2019-09-07 at 16 17 45](https://user-images.githubusercontent.com/14274852/64474831-66640100-d18b-11e9-8755-5735cb3ffb25.png)

**Comment by ihnatmoisieiev** [comment]
@Abedalkareem thank you! What about a placeholder text? Should it work the same?

**Comment by Abedalkareem** [comment]
Welcome, yes it's the same.
